spec/models/party_spec.rb

```diff
@@ -19,50 +19,74 @@
     it { should validate_uniqueness_of(:oscn_id) }
   end
 
-  describe '#without_birthday' do
-    it 'Filter to parties without birth month' do
-      party_bad = create(:party, birth_month: 12)
-      create(:party)
-      expect(described_class.without_birthday.size).to eq(1)
-      expect(described_class.without_birthday.first.id).not_to eq(party_bad.id)
+  describe 'scopes' do
```

Comment:
- moved existing specs under `scopes"`describe block
- added specs for `older_than` and `without_html`